packages/terra-toggle-section-header/tests/jest/ToggleSectionHeader.test.jsx

```diff
@@ -0,0 +1,49 @@
+import React from 'react';
+import ToggleSectionHeader from '../../src/ToggleSectionHeader';
+
+describe('ToggleSectionHeader', () => {
+  // Snapshot Tests
+  it('should render a default toggle section header', () => {
+    const wrapper = render(<ToggleSectionHeader title="default title">Test</ToggleSectionHeader>);
+    expect(wrapper).toMatchSnapshot();
+  });
+
+  it('should render an animated toggle section header', () => {
+    const toggleSectionHeader = render(<ToggleSectionHeader title="animated title">Test</ToggleSectionHeader>);
+    expect(toggleSectionHeader).toMatchSnapshot();
+  });
+
+  it('should render an initially open toggle section header', () => {
+    const toggleSectionHeader = render(<ToggleSectionHeader title="initially open title" isInitiallyOpen>Test</ToggleSectionHeader>);
+    expect(toggleSectionHeader).toMatchSnapshot();
+  });
+
+  // Event Tests
+  it('should render an open toggle section header when clicked', () => {
+    const toggleSectionHeader = mount(<ToggleSectionHeader title="opens on click title">opens on click section header</ToggleSectionHeader>);
+    toggleSectionHeader.find('SectionHeader').simulate('click');
+    expect(toggleSectionHeader).toMatchSnapshot();
+  });
+
+  // Prop Tests
+  it('should set sectionHeaderAttrs prop correctly', () => {
+    const sectionHeaderAttrProps = { backgroundColor: 'red' };
+    const toggleSectionHeader = shallow(<ToggleSectionHeader sectionHeaderAttrs={sectionHeaderAttrProps} title="sectionHeaderAttrs props test">Test</ToggleSectionHeader>);
+    expect(toggleSectionHeader.instance().props.sectionHeaderAttrs).toEqual(sectionHeaderAttrProps);
+    expect(toggleSectionHeader).toMatchSnapshot();
+  });
+
+  it('should set the children prop correctly', () => {
+    const children = 'test child';
+    const toggleSectionHeader = shallow(<ToggleSectionHeader title="sectionHeaderAttrs props test">{children}</ToggleSectionHeader>);
+    expect(toggleSectionHeader.instance().props.children).toEqual(children);
+    expect(toggleSectionHeader).toMatchSnapshot();
+  });
+
+  it('should set the level prop correctly', () => {
```

Comment:
We need to add jest tests for each level option

Reply:
Why? The point of the test is to see that supplying any `level` prop explicitly passed other than 2 is passed through to the underlying `terra-section-header`. Thus, I would suspect that passing any value other than `2` should be sufficient.

Reply:
^ Given your explanation above I'd say this is fine.